core/src/banking_stage.rs

```diff
@@ -82,6 +86,7 @@ impl BankingStage {
         verified_vote_receiver: CrossbeamReceiver<VerifiedPackets>,
         num_threads: u32,
     ) -> Self {
+        let batch_limit = TOTAL_BUFFERED_PACKETS / (num_threads as usize * PACKETS_PER_BLOB);
```

Comment:
One of the threads is only doing the vote processing, which should not have buffered packets. Maybe we can adjust batch_limit accounting for that..